app.py

```diff
@@ -108,7 +83,7 @@ def challenge5():
     if (os.getenv("PORT") == '8080' and re.search('(curl|wget)\/*', request.headers.get('User-Agent').lower())):
         return "Challenge String: Vjgfqgug it a qugwvy dopo vrql"
     else:
-        return pick_one(not_right)
+        return pick_one(not_right) #leaving this one as newline not required for webpage
```

Comment:
What do you mean by newline is not requred for webpage? This is challenge 5

Reply:
oops sorry got confused, thought that was something else. Please delete comment.